doc/examples/reconst_dki.py

```diff
@@ -211,20 +225,49 @@
 # biases in kurtosis parametric maps [Perron2015]_, [NetoHe2018]_.
 #
-# One can try to suppress this issue by using the more advance noise and
-# artefact suppression algorithms, e.g., as mentioned above, the MP-PCA
-# denoising (:ref:`sphx_glr_examples_built_preprocessing_denoise_mppca.py`)
+# One can try to suppress this issue by:
+# 1) using more advanced noise and artifact suppression algorithms, e.g.,
+# as mentioned above, the MP-PCA denoising
+# (:ref:`sphx_glr_examples_built_preprocessing_denoise_mppca.py`)
 # and Gibbs Unringing
 # (:ref:`sphx_glr_examples_built_preprocessing_denoise_gibbs.py`)
-# algorithms. Alternatively, one can overcome this artefact by computing the
-# kurtosis values from powder-averaged diffusion-weighted signals. The details
-# on how to compute the kurtosis from powder-average signals in dipy are
-# described in follow the tutorial
-# (:ref:`sphx_glr_examples_built_reconstruction_reconst_msdki.py`). Finally,
-# one can use constrained optimization to ensure that the fitted parameters
-# are physically plausible [DelaHa2020]_, as we will illustrate in the next
-# section. Ideally though, artefacts such as Gibbs ringing should be corrected
-# for as well as possible before using constrained optimization.
+# algorithms.
+# 2) computing the kurtosis values from powder-averaged diffusion-weighted
+# signals. The details on how to compute the kurtosis from powder-averaged
+# signals in DIPY are described in the following tutorial
+# (:ref:`sphx_glr_examples_built_reconstruction_reconst_msdki.py`).
+# 3) computing alternative definitions of mean and radial kurtosis such as
+# the mean kurtosis tensor (MKT) and radial tensor kurtosis (RTK) metrics (see
+# below).
+# 4) constrained optimization to ensure that the fitted parameters
+# are physically plausible [DelaHa2020]_ (see below).
+#
+# Alternative DKI metrics
+# =======================================================
+#
+# In addition to the standard mean, axial, and radial kurtosis metrics shown
+# above, alternative metrics can be computed from DKI, e.g.:
+# 1) the mean kurtosis tensor (MKT) - defined as the trace of the kurtosis
+# tensor - is a quantity that provides a contrast similar to the standard MK
+# but it is more robust to noise artifacts [Hansen2013]_, [Henriq2021]_.
+# 2) the radial tensor kurtosis (RTK) provides an alternative definition to
+# standard radial kurtosis (RK) that, as MKT, is more robust to noise
+# artifacts [Hansen2017]_.
+# 3) the kurtosis fractional anisotropy (KFA) that quantifies the anisotropy of
+# the kurtosis tensor [GlennR2015]_, which provides different information than
+# the FA measures from the diffusion tensor.
+# These measures are computed and illustrated below:
+
+compare_maps([dkifit], ['mkt', 'rtk', 'kfa'], fit_labels=['DKI'],
+             map_kwargs=[{'vmin': 0, 'vmax': 1.5}, {'vmin': 0, 'vmax': 1.5},
+                         {'vmin': 0, 'vmax': 1}],
+             filename='Alternative DKI metrics.png')
```

Comment:
Please, can you avoid space for filename. it can be messy  depending on the operating system. Use underscore